packages/node_modules/@node-red/nodes/core/core/80-function.js

```diff
@@ -158,9 +158,13 @@ module.exports = function(RED) {
             },
             env: {
                 get: function(envVar) {
-                    // For now, just return the env var. This will eventually
-                    // also return project settings and subflow instance properties
-                    return process.env[envVar]
+                    if (node && node._flow) {
+                        var flow = node._flow;
+                        if (flow) {
+                            return flow.getSetting(envVar);
+                        }
+                    }
+                    return process.env[envVar];
```

Comment:
I _think_ this should just return `node._flow.getSetting(envVar)`. 

 - `node` and `node._flow` will exist - no need to check for them. (If that breaks the tests, the tests need updating to ensure they properly initialise the node. If that is the case, come have a chat with me before changing anything)
 - `node._flow.getSetting` is responsible for getting the setting and falling back to `process.env` if needed - this node should do that for itself.

Reply:
Simplified according to your suggestion.